run_email_parse.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+import re,sys
+import scrapy
+from scrapy.crawler import CrawlerProcess
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+from scrapy import signals
+from scrapy.conf import settings
+from scrapy.xlib.pydispatch import dispatcher
+from selenium import webdriver
+
+settings.set('LOG_LEVEL', 'INFO')
```

Comment:
Log level for this should probably be ERROR, since most users will not care much about the INFO level stuff.